test/compact.jl

```diff
@@ -234,3 +190,13 @@ end
   end
 end
 
+
+@testset "Custom naming of @compact with NoShow" begin
+  _model = @compact(w=Dense(32, 32)) do x, y
+    tmp = sum(w(x))
+    return tmp + y
+  end
+  model = NoShow(_model) 
+  expected_string = "NoShow(...)         # 1_056 parameters"
+  @test similar_strings(get_model_string(model), expected_string)
```

Comment:
```suggestion
  @test similar_strings(get_model_string(model), expected_string)
  model2 = NoShow("test", _model)
  @test similar_strings(get_model_string(model), "test")
```
A simple test of custon naming

Reply:
Thanks!